edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/Constants.cs

```diff
@@ -3,7 +3,9 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Docker
 {
     static class Constants
     {
-        public const string EdgeAgentCreateOptionsName = "EdgeAgentCreateOptions";
+        public const string UnknownImage = "unknown";
```

Comment:
LGTM overall. Suggest changing the value to "unknown-module-docker-image" or something along those lines to make the logs more assertive and actionable.

Reply:
This doesn't really show up in the logs. This is a scenario when the EdgeAgent doesn't have its previous records. 
Also, I feel "unknown" is probably just as good as "unknown-module-docker-image" in conveying the info. Let me know if you disagree...